js/web/pages/[releasePubkey]/index.js

```diff
@@ -48,40 +48,40 @@ const ReleasePage = (props) => {
 
 export default ReleasePage
 
-export const getStaticPaths = async () => {
-  return {
-    paths: [
-      {
-        params: {
-          releasePubkey: 'placeholder',
-        },
-      },
-    ],
-    fallback: 'blocking',
-  }
-}
+// export const getStaticPaths = async () => {
```

Comment:
this was commented out due to that compiling bug on friday, should be resolved - not skipped